src/Custom/Chat/ChatFunctionChoice.Serialization.cs

```diff
@@ -13,6 +14,14 @@ void IJsonModel<ChatFunctionChoice>.Write(Utf8JsonWriter writer, ModelReaderWrit
 
     internal static void SerializeChatFunctionChoice(ChatFunctionChoice instance, Utf8JsonWriter writer, ModelReaderWriterOptions options)
     {
+#pragma warning disable SCME0001 // Type is for evaluation purposes only and is subject to change or removal in future updates.
```

Comment:
I wonder if we should just turn this warning off per type rather than try to be this focused.   Thoughts?

Reply:
The reason for doing it so is along the lines of principle of least privilege. If it were at the file level, users may not know what is necessitating this suppression.